### PR TITLE
Add Multiselect dragdrop Feature

### PR DESCRIPTION
**Pull request by joeizang**

### CODE REVIEW

**Comment by yann300** [comment]
After releasing in a drop zone, the modal doesn't sometimes show the right files name. 
Also when releasing it tries take in account the actual file that the mouse is hovering.

**Comment by joeizang** [comment]
in the file_explorer_multiselect.test.ts file, all tests pass on google chrome but one test, Should drag and drop multiple files in file explorer to tests folder #group1, fails on firefox. Can you look at this test @yann300 ?

File: libs/remix-ui/workspace/src/lib/css/file-explorer.css [hunk 1]
```diff
@@ -69,3 +69,7 @@ ul {
 .remixui_icons:hover {
     color: var(--text);
 }
+
+.remixui_selected {
```
Comment (ioedeveloper) [inline libs/remix-ui/workspace/src/lib/css/file-explorer.css:73]: Remove unused css class

Comment (joeizang) [reply libs/remix-ui/workspace/src/lib/css/file-explorer.css:73]: right. done.

File: libs/remix-ui/workspace/src/lib/components/flat-tree.tsx [hunk 2]
```diff
@@ -116,6 +124,12 @@ export const FlatTree = (props: FlatTreeProps) => {
 
   const onDragEnd = (event: SyntheticEvent) => {
     setIsDragging(false)
+    document.querySelectorAll('li.remixui_selected').forEach(item => {
```
Comment (ioedeveloper) [inline libs/remix-ui/workspace/src/lib/components/flat-tree.tsx:127]: Should be improved to use react hooks (useRef, useState, etc).

**Review by ioedeveloper — APPROVED** [review]

File: libs/remix-ui/workspace/src/lib/utils/getEventTarget.ts [hunk 3]
```diff
@@ -22,4 +22,35 @@ export const getEventTarget = async (e: any, useLabel: boolean = false) => {
...
+
+/**
+   *  When multiple files are selected in FileExplorer,
+   *  and these files are dragged to a target folder,
+   *  this function will build the profile of each selected item
+   *  in FileExplorer so they can be moved when dropped
+   * @param target - Initial target item in FileExplorer
+   * @returns - {DragStructure} Array of selected items
+   */
+export const buildMultiSelectedItemProfiles = (target: {
```
Comment (ioedeveloper) [inline libs/remix-ui/workspace/src/lib/utils/getEventTarget.ts:35]: Same with this function

**Pull request merged**